src/runtime_src/core/tools/xbutil2/XBDatabase.cpp

```diff
@@ -139,10 +175,55 @@ XBDatabase::create_complete_device_tree(boost::property_tree::ptree & _pt)
       CoreDevice.read_device_dma_stats(device_id, pt);
       ptPlatform.add_child("pcie_dma", pt);
     }
-
+#endif
+	{
+	  //boost::property_tree::ptree pt;
+	  XU_IP_LAYOUT *ipLayout = NULL;
```

Comment:
Please use structs from xclbin.h, that will make this code platform independent.